src/Employer/Employer.Web/Controllers/Part1/TitleController.cs

```diff
@@ -53,10 +69,27 @@ public async Task<IActionResult> Title(TitleEditModel m, [FromQuery] bool wizard
                 vm.PageInfo.SetWizard(wizard);
                 return View(vm);
             }
-
+            var currentVacancy = await _orchestrator.GetCurrentVacancy(m,response.Data);
```

Comment:
I don't think you need to do this here.

If `ReferredfromMA` TempData is set then all we need to do is do what the Dashboard does when it works out what route to use when its resuming a vacancy. So Once the vacancy has been created we can just find the next valid route and then redirect to there.